.github/workflows/_parse_version.yml

```
@@ -32,6 +32,9 @@ on:
       local:
         description: The local version
         value: ${{ jobs.parse.outputs.local }}
+      no_local:
+        description: The version without the local part
```

Is this a full version (including all previous fields) without the local part?
```suggestion
        description: The full version without the local part
```